connect/eaas/core/webapp_client.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import FastAPI
+from fastapi.staticfiles import StaticFiles
+from starlette.testclient import TestClient
+
+from connect.eaas.core.decorators import router
+
+
+class WebAppTestClient(TestClient):
+
+    def __init__(self, webapp, default_headers=None):
+        self._webapp_class = webapp
+        self.router = router
```

Comment:
Why set the router as a member of the class ?

Reply:
Removed